### PR TITLE
ON-10732 # Changed source to typescript

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: docs/form-elements/compliance.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+[Back to all Elements](./README.md)
+
+## Compliance (Checklist) Element
+
+Allow the user to select a single option from a list of options displayed as buttons and also optionally enter notes and add attachments.
+
+| Property       | Required | Type      | Default   | Description                                                                              |
+| -------------- | -------- | --------- | --------- | ---------------------------------------------------------------------------------------- |
+| `type`         | Yes      | `string`  | `'radio'` | The type of Form Element.                                                                |
```
Comment (Zaxist) [inline docs/form-elements/compliance.md:11]:
```suggestion
| `type`         | Yes      | `string`  | `'compliance'` | The type of Form Element.                                                                |
```

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "ES2020" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```
Comment (Zaxist) [inline tsconfig.json:7]: IMO for a node SDK it's still a good idea to just target ES5

Comment (mymattcarroll) [reply tsconfig.json:7]: We are only supporting node >=10 

We don't need to compile before ES2020 as far as I know.

Comment (mymattcarroll) [reply tsconfig.json:7]: nope I have that wrong, we want ES2018

Comment (Zaxist) [reply tsconfig.json:7]: yes 2020, will compile with `import/export` syntax which won't work in node 10

Comment (mymattcarroll) [reply tsconfig.json:7]: `"module": "commonjs"` will make it compile to `module.exports` and `require()` but there are other syntax changes that will not be supported but node 10.

Changing now.

**Pull request merged**